spanner/client_test.go

```diff
@@ -960,6 +960,35 @@ func TestClient_ReadWriteTransaction_DoNotLeakSessionOnPanic(t *testing.T) {
 	}
 }
 
+func TestClient_SessionContainsDatabaseRole(t *testing.T) {
+	// Make sure that there is always only one session in the pool.
+	sc := SessionPoolConfig{
+		MinOpened: 1,
+		MaxOpened: 1,
+	}
+	server, client, teardown := setupMockedTestServerWithConfig(t, ClientConfig{SessionPoolConfig: sc, DatabaseRole: "test"})
+	defer teardown()
+
+	// Wait until all sessions have been created, so we know that those requests will not interfere with the test.
+	sp := client.idleSessions
+	waitFor(t, func() error {
+		sp.mu.Lock()
+		defer sp.mu.Unlock()
+		if uint64(sp.idleList.Len()) != 1 {
+			return fmt.Errorf("num open sessions mismatch.\nWant: %d\nGot: %d", sp.MinOpened, sp.numOpened)
```

Comment:
Standard unit test terminology is 'Expected' and 'Actual' (with Expected mentioned before Actual) instead of Got/Want, but I see existing tests are already using got/want, so this is fine.

Existing tests have also consistently specified Got before want. Let's do the same for consistency.

Reply:
Sorry to wade in on this PR, but I feel that this one is a little too important to ignore: The idiomatic naming for testing actual/expected in Go is actually `got, want := ...`, and not `actual, expected := ...`.

The point about `Got` before `Want` is valid.

Reply:
I actually copied this line from a different test case in this file. But I agree, we should be consistent with other similar checks.